### ISSUE TITLE
Provide a monitoring interface for circuit-breakers

### ISSUE THREAD

**Issue by kubamarchwicki** [spring-cloud-incubator/spring-cloud-circuitbreaker#34]
To be able to monitor the circuit breakers effectively it would be good to expose their state (closed, open, half-open) as of Spring Actuator endpoints without additional effort.

Monitoring the state of circuit breakers would allow discovering that a downstream dependency suddenly becomes unavailable. It is essential for applications used in production.

**Comment by ryanjbaxter** [comment]
Are we talking about monitoring just in an actuator sense?

**Comment by kubamarchwicki** [comment]
Yes. At least the current state (as observed: OPEN / CLOSED), possibly last flip time, maybe some statistics - depending on data availability for any given implementation of circuit breakers.

**Comment by kubamarchwicki** [comment]
Depending on timelines, I should be able to commit some time to that.
